### ISSUE TITLE
Replace (Ctrl+H) doesn't copy selected text in the Search box

### ISSUE THREAD

**Issue by RistoVaaraniemi** [wbsoft/frescobaldi#833]
Hi,

I noticed that when I try to search & replace text (`Ctrl+H`) the selected text is not copied in the Search box. If I press `Ctrl+F` to search stuff the selected text is copied. I'd expect them to behave the same way.

I don't remember if this is a new issue or if it has been like this from the start.

BR,
Risto

**Comment by wbsoft** [comment]
It has been designed as this: often you want to perform a replacement in a selected region, that's why the selection is not copied when the search entry is opened in replace mode.

**Comment by RistoVaaraniemi** [comment]
OK. That explains the behaviour. I hadn't tested replacing in a selection. I usually have to do it globally so this doesn't help me much --> lot of copying & pasting & clicking.

Would it make any sense to have a "Replace in the selection" checkbox that would allow the user to select the area where the replacing should be performed?

-Risto

**Comment by wbsoft** [comment]
Yes, that would make sense, although the button "All" already does that when there is a selection.
